Author: Michael Karcher <debian@mkarcher.dialup.fu-berlin.de>
Description: This patch makes unace use utf8 filenames on linux instead of
 Latin1.

--- unace-nonfree-2.5.orig/source/base/all/arcblk/arcblk.c
+++ unace-nonfree-2.5/source/base/all/arcblk/arcblk.c
@@ -282,12 +282,8 @@ void    BASE_ARCBLK_ConvertHeader32ToHea
 #if defined(__MACOSX__)
     BASE_ARCBLK.Header.File.FNAME_SIZE = BASE_UNINORM_CP850ToUTF8NFD(
       BASE_ARCBLK.Header.File.FNAME, BASE_ARCBLK.Header.File.FNAME_SIZE);
-#elif defined(__LINUX__)
-    /* may not work for every linux user */
-    BASE_ARCBLK.Header.File.FNAME_SIZE = BASE_UNINORM_CP850ToANSI(
-      BASE_ARCBLK.Header.File.FNAME, BASE_ARCBLK.Header.File.FNAME_SIZE);
 #elif defined(__unix__)
-    /* for every other unix assume UTF-8 NFC normalization */
+    /* for unix, assume UTF-8 NFC normalization */
     BASE_ARCBLK.Header.File.FNAME_SIZE = BASE_UNINORM_CP850ToUTF8NFC(
       BASE_ARCBLK.Header.File.FNAME, BASE_ARCBLK.Header.File.FNAME_SIZE);
 #endif
